llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -244,6 +244,9 @@ def FeatureExperimentalZeroingPseudos
 def FeatureUseScalarIncVL : SubtargetFeature<"use-scalar-inc-vl",
   "UseScalarIncVL", "true", "Prefer inc/dec over add+cnt">;
 
+def FeatureUseFixedIfEqualToScalable : SubtargetFeature<"use-fixed-if-equal-to-scalable",
```

Comment:
This could probably do with the word "Cost" in their somewhere. Maybe FeatureUseFixedOverScalableIfEqualCost

Reply:
Does this need to be a full-blown feature if you intend to just enable it by default for V2 anyway? When we added VScaleForTuning, we just used a boolean in AArch64Subtarget that gets set during initialisation for the particular CPU. See AArch64Subtarget::initializeProperties.

Reply:
Ah ok. Fair enough!

Reply:
In that case maybe it's worth us revisiting VScaleForTuning as well and making that a feature for consistency.

Reply:
Thanks for the naming suggesting, I was struggling with the name, will change it to `FeatureUseFixedOverScalableIfEqualCost`.